pdns/recursordist/docs/changelog/4.1.rst

```diff
@@ -1,6 +1,24 @@
 Changelogs for 4.1.x
 ====================
 
+.. changelog::
+  :version: 4.1.11
+  :released: 1st of February 2019
+
+  Since Spectre/Meltdown, system calls have become more expensive.  This caused performance issues in the PowerDNS Recursor.
```

Comment:
That seems a bit generic, perhaps "This made exporting a very high number of protobuf messages costly, which is addressed in this release by reducing the number of sycalls per message", or something like that?